ferretdb/ferretdb.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn"
+	"github.com/FerretDB/FerretDB/internal/handlers/registry"
+	"github.com/FerretDB/FerretDB/internal/util/logging"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/version"
+)
+
+// Config contains a PostgreSQL backend connection string.
+type Config struct {
+	PostgresURL string
+	TigrisURL   string
+}
+
+// FerretDB proxy.
+type FerretDB struct {
+	config   Config
+	mongoURL string
+}
+
+// New returns a new FerretDB.
+func New(conf Config) FerretDB {
```

Comment:
Why do we pass config to both New and Run?

Reply:
> Why do we pass config to both New and Run?

changed to one place.